pkg/controller/plan/kubevirt.go

```diff
@@ -85,6 +87,12 @@ const (
 	qemuGroup = int64(107)
 )
 
+// Preference configmaps names
+const (
+	vsphereOsConfigMap = "forklift-vsphere-osmap"
+	ovirtOsConfigMap   = "forklift-ovirt-osmap"
```

Comment:
how about getting these values from the operator using environment variables?

Reply:
i think this will resolve the build-in mappings to be configurable :)